atlp-rwanda/Team-furebo-E-commerce-fn
# #185172186 A user should be able to update an item in their collection

### What does this PR do?
This PR adds the update product functionality
### Description of Task to be completed
A user, in this case, a seller will be able to keep his inventory up to date according to their encumen. Items quantity, prices, availability, and so on would be adjustable.
### How should this be manually tested?
* Navigate to the login page and click and sign in, if confirmed the user is a seller they have an option of viewing their collection.
* From the collection a single product can be accessed, and all the properties of that particular product will be displayed and they will be able to update them via a button present on the page that calls a form.
* The form can be filled with new values and the not needed one will remain as is.
* Enter a new password and confirm it, make sure it complies with the validation terms, and Submit.
### What are the relevant pivotal tracker/Trello stories?
#185172186
